core/common/src/main/java/alluxio/Constants.java

```diff
@@ -120,7 +121,7 @@ public final class Constants {
   public static final String METRICS_MASTER_CLIENT_SERVICE_NAME = "MetricsMasterClient";
   public static final String BLOCK_WORKER_CLIENT_SERVICE_NAME = "BlockWorkerClient";
   public static final String FILE_SYSTEM_WORKER_CLIENT_SERVICE_NAME = "FileSystemWorkerClient";
-  public static final String JOURNAL_MASTER_CLIENT_SERVICE_NAME = "JournalMaster";
```

Comment:
will this change cause any backward compatibility issues? e.g. in HA mode doing a rolling upgrade, or using old client connecting to the new masters.

Reply:
When investigating its usages, I only saw it being used in log messages or error messages. I could not find a usage that affected the behavior of the client or server. Let me know if you find such a case.